lib/utils.py

```diff
@@ -126,3 +126,14 @@ def run_command(cmd, **kwargs):
                                         stdout=output, stderr=error_output)
 
     return output
+
+
+def replace_str_in_file(file_path, search, replacement):
+
+    lines = []
+    with file(file_path, "r") as f:
+        lines = f.readlines()
+    with file(file_path, "w") as f:
+        for line in lines:
+            line = line.replace(search, replacement)
+            f.write(line)
```

Comment:
Using fileinput reduces code size, but I personally dislike the print() in it. I had to open Python std lib doc to understand that "the file is moved to a backup file and standard output is directed to the input file" in it. To me, the code above is simpler to understand. But if you guys prefer using fileinput, I can use it.